infrastructure/mongodb/create_db.js

```diff
@@ -0,0 +1,5 @@
+#!/bin/bash
+
+// MongoDB is fundamentally designed for "create on first use"
+// If you do not insert data with your JavaScript files, then no database is created.
+db.createCollection("default");
```

Comment:
Could we also use planthor- instead default? but It's okay

Reply:
sure we can

Reply:
updated